.github/workflows/publish.yml

```
@@ -0,0 +1,48 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - 'DOI-759-ev-dashboard-client-npm-publish'
+  pull_request:
+    branches:
+      - '*'
+jobs:
+  cancel-previous:
+    name: 'Cancel Previous Runs'
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+  publish-packages:
+    runs-on: ubuntu-latest
+    needs: cancel-previous
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+          registry-url: https://registry.npmjs.org/
+
```

Could we add a call to rush change -v in here to make sure that the change files have been generated?
```suggestion
- name: Rush change verification
        run: node common/scripts/install-run-rush.js change -v
```

---

@royki Added this here: https://github.com/energywebfoundation/ev-dashboard-client/commit/65fee32b4d53985e9479c86e660bd6ad86f610cd